openapi/paths/tracking@taxes@{id}.yaml

```
@@ -0,0 +1,27 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+get:
+  x-products:
+    - Users
+  tags:
+    - Tracking
+  summary: Retrieve a tracking tax log
+  operationId: GetTaxLog
+  x-sdk-operation-name: getTaxLog
```

nitpick: we may be missing a description here also 
```suggestion
  x-sdk-operation-name: getTaxLog
  descriptions: Retrieves a tracking tax log with a specified ID.
```

---

Should be `description`, not `descriptions` though.

---

Thanks for spotting this 😅 Updated my suggestion to `description`